### PR TITLE
service card component

### LINKED ISSUE #37: service-card-service-page component
## Common Component - service-card-service-page :
It will be on the service page and provide more information about the service.
- props: data, variant.

- [ ]  It will be used in Pages :
Service page.

- [ ]  Material UI link :
[Card link](https://material-ui.com/components/cards/)

- [ ] UI Image From Design :

image

### PR DESCRIPTION
**Pull request by yasmeenattallah**
Fix #37

### CODE REVIEW

**Review by imansedky5 — DISMISSED** [review]

**Review by khamis-hesham — DISMISSED** [review]

**Review by mFlaifel — COMMENTED** [review]
please increase the component sized to render the description correctly.

**Review by khamis-hesham — DISMISSED** [review]

File: client/src/component/serviceCard/index.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  const fakeImage =
+    'https://www.matrix.com/~/media/images/hair-color-gallery/hair-color-looks/highlights/brown-highlights/brown-highlights-1.jpg';
+  return (
+    <Card className={classes.card}>
+      <CardActionArea className={classes.root}>
+        <CardMedia>
+          <img className={classes.image} src={img || fakeImage} alt="service" />
+        </CardMedia>
+        <CardContent>
+          <Typography gutterBottom variant="h5" component="h2">
```
Comment (AdhamHaisami) [inline client/src/component/serviceCard/index.js:25]: why variant and component are different?

File: client/src/component/serviceCard/index.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            {name}
+          </Typography>
+          <Typography variant="body2" color="textSecondary" component="p">
+            {description.split(' ').splice(0, 11).join(' ')}
+          </Typography>
+        </CardContent>
+        <CardActions>
+          <Link to={`${BOOK}`} className={classes.link}>
```
Comment (AdhamHaisami) [inline client/src/component/serviceCard/index.js:33]: What is the purpose of using backtick while there is no string around?

**Review by AdhamHaisami — APPROVED** [review]

File: client/src/component/serviceCard/index.js (hunk 1 above)
Comment (yasmeenattallah) [reply client/src/component/serviceCard/index.js:25]: check [https://material-ui.com/api/typography/](url)
component is the element type 
but varient is for style you con make them different or same according to ur requirement

**Review by imansedky5 — DISMISSED** [review]

**Pull request merged**